operators/bosh-operator/DirectorService.js

```diff
@@ -742,7 +742,7 @@ class DirectorService extends BaseDirectorService {
       this.executeActions(CONST.SERVICE_LIFE_CYCLE.PRE_BIND, actionContext),
       this.getDeploymentIps(deploymentName),
       (preBindResponse, ips) => utils.retry(() => this.agent.createCredentials(ips, binding.parameters, preBindResponse), {
-        maxAttempts: 3,
+        maxAttempts: 2,
```

Comment:
Our intention was also to not show operation failed after 3(2) attempts only as message and also show specific message. How would that be achieved ?

Reply:
How does it look now?

Reply:
cool!